doc/ubuntu_release_process.md

```diff
@@ -115,42 +117,53 @@ Last, we need to push our tag above.
 
 
 ### cherry-picked changes ###
-This is generally not the mechanism that is preferred for any release supported by trunk.  It may be used in order to get a upload in *really quickly* for a hot fix.
+This is generally not the preferred mechanism for any release supported by trunk.  It is used either to get a upload in *really quickly* for a hot fix or during release **Feature Freeze** when only bugfixes or FeatureFreezeExceptions (FFEs) are accepted.  The tool for doing this is in ``uss-tableflip/scripts/cherry-pick``.  It takes as import a commit-ish that it will create a patch to apply the commitish via debian/patches/cpick-\*.
 
 #### Cherry-pick Process
-The tool for doing this is in ``uss-tableflip/scripts/cherry-pick``.  It takes as import a commit-ish that it will create a cherry-pick from.
-
-    ## just tag current so you have a revert point.
-    $ git tag --delete xx-current >/dev/null 2>&1; git tag xx-current
-    $ git checkout ubuntu/xenial
-    $ cherry-pick dc2bd79949
-
-The tool will:
-
-  * add a new file to debian/patches/ named cpick-<commit>-topic that is dep-8 compatible
-  * add that patch to debian/patches/series
-  * verify that the patch applies (quilt push -a)
-  * give you a chance to edit formatted the debian/changelog entry
-  * prompt you to commit the change.
+```
+  $ git fetch upstream; git checkout upstream/ubuntu/xenial -b ubuntu/xenial
```

Comment:
Let's not collapse multi lines into a single one.  Also if users already have a local ubuntu/xenial then they'll need the git reset --hard upstream/ubuntu/xenial.

```
$ git fetch upstream
$ git checkout -b ubuntu/xenial upstream/ubuntu/xenial || git checkout ubuntu/xenial
$ git reset --hard upstream/ubuntu/xenial 
```